### PR TITLE
Get going with new S3 backend

### PR DESCRIPTION
**Pull request by FynnBe**

### CODE REVIEW

File: .github/workflows/stage_call.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
           python-version: "3.12"
           cache: "pip" # caching pip dependencies
       - run: pip install -r requirements.txt
-      - name: stage
```
Comment (jmetz) [inline .github/workflows/stage_call.yaml:47]: Is the switch from `name` to `id` here accidental from our recent refactor of "model-name" to "resource-id"?

File: scripts/upload_model_to_zenodo.py [hunk 2]
```diff
@@ -8,14 +9,13 @@
 from typing import Optional
 from urllib.parse import quote_plus, urljoin, urlparse
 
-import requests  # type: ignore
```
Comment (jmetz) [inline scripts/upload_model_to_zenodo.py:11]: I guess you don't get mypy errors about these modules 😅?

Or did you update to typed versions?

**Review by jmetz — APPROVED** [review]
Overall lgtm, though haven't explicitly tested (as I don't have local testing setup for full CI workflow)

File: .github/workflows/stage_call.yaml (hunk 1 above)
Comment (FynnBe) [reply .github/workflows/stage_call.yaml:47]: no, intentional as its outputs are used

Comment (FynnBe) [reply .github/workflows/stage_call.yaml:47]: (and while chaning I thought the name is just hiding the python script call)

**Pull request merged**